packages/sources/layer2-sequencer-health/src/config/index.ts

```diff
@@ -26,9 +38,21 @@ const DEFAULT_OPTIMISM_RPC_ENDPOINT = 'https://mainnet.optimism.io'
 const DEFAULT_METIS_RPC_ENDPOINT = 'https://andromeda.metis.io/?owner=1088'
 
 export const RPC_ENDPOINTS = {
-  [Networks.Arbitrum]: util.getEnv('ARBITRUM_RPC_ENDPOINT') || DEFAULT_ARBITRUM_RPC_ENDPOINT,
-  [Networks.Optimism]: util.getEnv('OPTIMISM_RPC_ENDPOINT') || DEFAULT_OPTIMISM_RPC_ENDPOINT,
-  [Networks.Metis]: util.getEnv('METIS_RPC_ENDPOINT') || DEFAULT_METIS_RPC_ENDPOINT,
+  [Networks.Arbitrum]: util.getEnv(ENV_ARBITRUM_RPC_ENDPOINT) || DEFAULT_ARBITRUM_RPC_ENDPOINT,
```

Comment:
Not blocking approval, but can you explain this bracket syntax?

Reply:
I believe it's called Dynamic Property Key syntax. Networks is enum value (can be another object). Instead of `{'metis': '1088'}` we can use `{[Networks.Metis]: '1088'}` both are the same but second one in this particular case makes more sense because there are multiple places where `metis` is used as an object key .

Reply:
Thanks for teaching me!